SQLClient/src/internal/queryMaker/queryMaker.go

```diff
@@ -82,6 +82,30 @@ func (q *QueryMaker) GetVotesForCandidate(candidate, county string, states []str
 	return byteToInt, query, nil
 }
 
+func getStringForGetCountyAnnotations(state string, county string) string {
+	qString := fmt.Sprintf("select state, county, annotations from County where annotations != \"\"")
+
+	if state != ""{
+		qString = fmt.Sprintf("%s and state=%q", qString, state)
+	}
+	if county != "" {
+		qString = fmt.Sprintf("%s and county=%q", qString, county)
+	}
+	return qString
+}
+
+func (q *QueryMaker) GetCountyAnnotations(state string, county string) ([]map[string]interface{}, []string, string, error) {
+	query := getStringForGetCountyAnnotations(state, county)
+	rows, colNames, err := q.DoRawQuery(query)
+	if err != nil {
+		return nil, nil, "", err
+	}
+	if len(rows) < 1 || rows[0][colNames[0]] == nil {
+		return nil, nil, "", errors.New("could not find any matches")
```

Comment:
this should still return the string query so we can log it from the caller

Reply:
LOL this is literally one of the 2 bugs I fixed earlier and I just added another one. Good catch LOL tyty

Reply:
You can notice on this branch that the same bug is in GetVotesForCandidate which I mindlessly copied which is why its here as well LOL